### ISSUE TITLE
Error on `rounded` prototype when breakpoint is enabled

### ISSUE THREAD

**Issue by aHxShUM** [zurb/foundation-sites#10333]
#### How to reproduce this bug:

1. `settings.scss`
```scss
$global-prototype-breakpoints: true;
```

2. Step two
```scss
@include foundation-everything($prototype: true);
```

#### What happened instead:
```
            @include rounded-radius;
                    ^
      No mixin named rounded-radius
```

#### Foundation Version(s) you are using:
6.4.1

**Comment by IamManchanda** [comment]
Thanks for the bug report @aHxShUM

**Comment by IamManchanda** [comment]
Hey @aHxShUM This is fixed #10335 

@rafibomb I think when you changed component styling classes ... you forgot this responsive class maybe but now fixed 

@rafibomb @kball Can we get a patch version of this release soon (of course with some xy bug solved) :wink:
